### PR TITLE
✨ Add building-list to extensions at dashboard

### PR DESCRIPTION
**Pull request by JRH-1997**
This will create a building list for each extension in the Building overview of the Dashboard. 

image
image

### CODE REVIEW

File: src/modules/dashboard/components/building-list.vue [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        >
+            <tr v-for="building in buildingsSorted" :key="building.id">
+                <td>
+                    <a class="btn btn-default btn-xs disabled">
+                        {{ buildingTypeNames[building.building_type] }}
+                    </a>
+                </td>
+                <td>
+                    <a
+                        :href="`/buidlings/${building.id}`"
```
Comment (Suchty112) [inline src/modules/dashboard/components/building-list.vue:27]: buidlings to buildings :D

Comment (JRH-1997) [reply src/modules/dashboard/components/building-list.vue]: Thanks @Suchty112

**Comment by Suchty112** [comment]
Jan came up with the idea of doing this for the buildings in general as well.
Or if you have a lot of time, list them all in the individual categories.

For the first, here is an example. 
https://cdn.discordapp.com/attachments/747838946343518318/795411620498899014/unknown.png
Only the icon then directly behind the name and not as on the screenshots.

**Comment by JRH-1997** [comment]
> Jan came up with the idea of doing this for the buildings in general as well.
> Or if you have a lot of time, list them all in the individual categories.
> 
> For the first, here is an example.
> https://cdn.discordapp.com/attachments/747838946343518318/795411620498899014/unknown.png
> Only the icon then directly behind the name and not as on the screenshots.

Done: 
image
image

**Comment by JRH-1997** [comment]
fastDispatchChooser is added to for all buildingTypes except the DispatchCenter. 
I didn't add it for the extensions. I don't think it is needed there. If you want. I could add that.

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**